xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/widgets/fullScreen.css

```diff
@@ -67,6 +62,9 @@
 #body .fullScreenWrapper .btn {
   width: auto;
 }
+#body .fullScreenWrapper .buttons .buttonwrapper input {
```

Comment:
can you reduce this specificity? lots of classes ... are they all needed?

Reply:
I've copied that from the above line, but it can totally be reduced with
```css
.fullScreenWrapper .buttonwrapper {
```